### PR TITLE
[docs] Add more information to readme

### PR DESCRIPTION
**Pull request by An-prog-hub**

### CODE REVIEW

**Review by oliviertassinari — APPROVED** [review]
Great idea. I have updated it to be closer to the readme of the main repository. We can't cover the data grid component specifically as this repository isn't only about the component, but all the other ones we are bringing.

File: README.md [hunk 1]
```diff
@@ -1,7 +1,38 @@
...
+
+## Questions
+
+For _how-to_ questions and other non-issues,
+please use [StackOverflow](https://stackoverflow.com/questions/tagged/material-ui) instead of GitHub issues.
+There is a StackOverflow tag called "material-ui" that you can use to tag your questions.
+
+## Examples
+
+Are you looking for an example project to get started?
```
Comment (mbrookes) [inline README.md:21]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -1,7 +1,38 @@
...
+[We host some](https://material-ui.com/getting-started/example-projects/).
```
Comment (mbrookes) [inline README.md:22]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -1,7 +1,38 @@
...
 - [DataGrid](https://material-ui.com/components/data-grid/)
+
+## Questions
+
+For _how-to_ questions and other non-issues,
+please use [StackOverflow](https://stackoverflow.com/questions/tagged/material-ui) instead of GitHub issues.
+There is a StackOverflow tag called "material-ui" that you can use to tag your questions.
+
+## Examples
+
```
Comment (mbrookes) [inline README.md:20]:
```suggestion
```

File: README.md [hunk 4]
```diff
@@ -1,7 +1,38 @@
...
 
 - [DataGrid](https://material-ui.com/components/data-grid/)
+
+## Questions
+
+For _how-to_ questions and other non-issues,
+please use [StackOverflow](https://stackoverflow.com/questions/tagged/material-ui) instead of GitHub issues.
+There is a StackOverflow tag called "material-ui" that you can use to tag your questions.
+
+## Examples
```
Comment (mbrookes) [inline README.md:19]:
```suggestion
```

File: README.md [hunk 5]
```diff
@@ -1,7 +1,38 @@
...
+
+Are you looking for an example project to get started?
+[We host some](https://material-ui.com/getting-started/example-projects/).
+
```
Comment (mbrookes) [inline README.md:23]:
```suggestion
```

File: README.md [hunk 6]
```diff
@@ -1,7 +1,38 @@
...
+## Contributing
+
+Read our [contributing guide](/CONTRIBUTING.md) to learn about our development process, how to propose bugfixes and improvements, and how to build and test your changes to Material-UI.
+
+Notice that contributions go far beyond pull requests and commits.
+Although we love giving you the opportunity to put your stamp on Material-UI, we also are thrilled to receive a variety of [other contributions](https://material-ui.com/getting-started/faq/#material-ui-is-awesome-how-can-i-support-the-project).
+
+## Changelog
+
+Recently Updated?
```
Comment (mbrookes) [inline README.md:33]:
```suggestion
```

File: README.md [hunk 7]
```diff
@@ -1,7 +1,38 @@
...
+Please read the [changelog](https://github.com/mui-org/material-ui-x/releases).
```
Comment (mbrookes) [inline README.md:34]:
```suggestion
If you have recently updated, please read the [changelog](https://github.com/mui-org/material-ui-x/releases) for details of what has changed.
```

File: README.md [hunk 8]
```diff
@@ -1,7 +1,38 @@
...
 ## Components
 
 - [DataGrid](https://material-ui.com/components/data-grid/)
+
+## Questions
+
+For _how-to_ questions and other non-issues,
+please use [StackOverflow](https://stackoverflow.com/questions/tagged/material-ui) instead of GitHub issues.
+There is a StackOverflow tag called "material-ui" that you can use to tag your questions.
+
```
Comment (mbrookes) [inline README.md:18]: Should we link to the paid version with reference to the included support?

**Review by mbrookes — CHANGES_REQUESTED** [review]
Linking to the basic examples doesn't make much sense for the advanced components.

File: README.md (hunk 8 above)
Comment (oliviertassinari) [reply README.md:18]: The "Pro" plan, the only one we sell yet, has no dedicated support attached to it.

Comment (oliviertassinari) [reply README.md:18]: But maybe we can clarify it here, yes +1

Comment (mbrookes) [reply README.md:18]: > The "Pro" plan, has no dedicated support attached to it

According to the wireframe it comes with a year of support.

File: README.md [hunk 9]
```diff
@@ -16,11 +16,6 @@ For _how-to_ questions and other non-issues,
...
 There is a StackOverflow tag called "material-ui" that you can use to tag your questions.
 
-## Examples
-
-Are you looking for an example project to get started?
-[We host some](https://material-ui.com/getting-started/example-projects/).
-
 ## Contributing
 
 Read our [contributing guide](/CONTRIBUTING.md) to learn about our development process, how to propose bugfixes and improvements, and how to build and test your changes to Material-UI.
```
Comment (mbrookes) [inline README.md:21]:
```suggestion
Read the [contributing guide](/CONTRIBUTING.md) to learn about our development process, how to propose bugfixes and improvements, and how to build and test your changes to Material-UI X.
```

File: README.md (hunk 8 above)
Comment (oliviertassinari) [reply README.md:18]: I was referring to:

Capture d’écran 2020-11-05 à 21 34 09

in which I believe we provide support as GitHub issues :)

Comment (mbrookes) [reply README.md:18]: So was I, but let's park it for another time.

Comment (oliviertassinari) [reply README.md:18]: Considering Pro have priority on GitHub, I 👍  the original comment.

**Pull request merged**